deploy/crds/app.kiegroup.org_kogitodataindices_crd.yaml

```diff
@@ -171,25 +171,33 @@ spec:
               description: Status of the Data Index Service Deployment created and
                 managed by it
               properties:
+                availableReplicas:
+                  description: Total number of available pods (ready for at least
+                    minReadySeconds) targeted by this deployment.
+                  format: int32
+                  type: integer
                 collisionCount:
```

Comment:
Is this field still needed?

Reply:
Those are auto generated by the Deployment Status type, there's not much to do unfortunately.